src/_includes/contributor/rewards.md

```diff
@@ -64,8 +77,8 @@ Every merged PR to the [DevDocs repository](https://github.com/magento/devdocs)
 
 Achievement | Points | Description
 | ------------ | --- | --- |
-Improvement | 10 | Contribution contains document improvements, adding missing features of inconsistency between code base and documentation, and so on.
-Editorial | 1 | Contribution contains fixes only for typos, grammatical inconsistencies, or minor rewrites to correct inaccuracies.
+Improvement | 10 | Contribution contains document improvements, adding missing features of inconsistency between code base and documentation, and similar.
```

Comment:
"Improvement" is not a tag...at least any more...at least not for Devdocs. But I would love something like "Minor Update", between Editorial and Major. More than typo fix but below a new heading/section.

Reply:
Yeah, since our contribution points are managed by EngComm I will have to confirm if "Improvement" is a label for them.

And this PR is def not about creating _new_ labels like "minor update". 🤣

Reply:
Update: I think Improvement is simply the name of the achievement to incur points, it may not be directly related to an "improvement" label. @VladimirZaets can you confirm?

Reply:
@shrielenee yes, correct.
"Improvement" it’s abstract label for good quality PR. 
It’s like PR type, in case of DevDocs type can be Improvement or Editorial. One of those two labels is required to be added to any PR. Additionally, we can reward contributors with labels like “Technical” and contributor will receive additional points. It’s like a kind of booster.